Sources/Mailgun/Models/TemplateMessage.swift

```diff
@@ -0,0 +1,89 @@
+import Vapor
+
+extension Mailgun {
+    public struct TemplateMessage: Content {
+        public static var defaultContentType: MediaType = MediaType.formData
+        
+        public typealias FullEmail = (email: String, name: String?)
+        
+        public let from: String
+        public let to: String
+        public let replyTo: String?
+        public let cc: String?
+        public let bcc: String?
+        public let subject: String
+        public let template: String
+        public let templateData: [String:String]?
+        public let templateVersion: String?
+        public let templateText: Bool?
+        public let attachment: [File]?
+        public let inline: [File]?
+        
+        private enum CodingKeys : String, CodingKey {
+            case from, to, replyTo = "h:Reply-To", cc, bcc, subject, template, templateData = "h:X-Mailgun-Variables", templateVersion = "t:version", templateText = "t:text", attachment, inline
```

Comment:
Would be better to have each case on separate line at least to stay in the same code style with other files.
In the future when some CodingKey may be changed it would be hard to recognize what exactly changed if leave all coding keys declared in one line.